src/main/java/com/noticemedan/mappr/viewmodel/CanvasView.java

```diff
@@ -436,8 +446,8 @@ public void toggleDijkstraNetwork() {
 	}
 
 	public void toggleRandomShortestPath() {
-		this.showRandomSP = !this.showRandomSP;
+		this.showPath = !this.showPath;
```

Comment:
Hvorfor navneændring her? this.showRandomSP skulle gerne være reserveret til at toggle random SP alene.

Reply:
Jeg tænkte da det var en udvikler feature, at den ikke skulle vises sammen med en reel SP, så nu overskriver den i stedet